connectors/src/connectors/github/temporal/workflows.ts

```diff
@@ -118,6 +125,70 @@ export async function githubFullSyncWorkflow(
   await githubSaveSuccessSyncActivity(dataSourceConfig);
 }
 
+export async function githubSyncAllReposWorkflow(
+  dataSourceConfig: DataSourceConfig,
+  connectorId: ModelId,
+  // Used to re-trigger a code-only full-sync after code syncing is enabled/disabled.
+  syncCodeOnly: boolean,
+  forceCodeResync = false
+) {
+  await githubSaveStartSyncActivity(dataSourceConfig);
+
+  const queue = new PQueue({ concurrency: MAX_CONCURRENT_REPO_SYNC_WORKFLOWS });
```

Comment:
any reason why we don't use the `concurrentExecutor` here?
If the reason was to not change the workflow code, now is a good time

Reply:
any idea why was https://github.com/dust-tt/dust/pull/3379 reverted?

Reply:
1. workflows are executed by temporal servers, we use as little libs as possible since it can break it (but maybe it'd work
2. I remember somebody (flavien I think) implemented, then went back, then implemented again but couldn't say why

Reply:
AFAICT concurrentExecutor is only a thin layer on top of p-queue so that's really be the same here, only less code and we'd be using the cool stuff other eng made for the sake of our happiness

Reply:
but well, as seen IRL let's not bother with this, we'll deal with this separately if we want to (I'm not sure I really really do :p)